src/OrchardCore.Modules/OrchardCore.ArchiveLater/Drivers/ArchiveLaterPartDisplayDriver.cs

```diff
@@ -49,7 +49,7 @@ public override async Task<IDisplayResult> UpdateAsync(ArchiveLaterPart part, IU
 
             await updater.TryUpdateModelAsync(viewModel, Prefix);
 
-            if (viewModel.ScheduledArchiveLocalDateTime == null || httpContext.Request.Form["submit.Save"] == "submit.CancelArchiveLater")
+            if (viewModel.ScheduledArchiveLocalDateTime == null || httpContext.Request.Form["submit.Publish"] == "submit.CancelArchiveLater")
```

Comment:
If we do so, we need to change this too

https://github.com/OrchardCMS/OrchardCore/blob/242115849b741325b667050b40b78d947ea477f0/src/OrchardCore.Modules/OrchardCore.PublishLater/Drivers/PublishLaterPartDisplayDriver.cs#L57